src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1442,13 +1442,23 @@ protected void AfterMembersChecks(DiagnosticBag diagnostics)
                 }
 
                 // https://github.com/dotnet/roslyn/issues/30080: Report diagnostics for base type and interfaces at more specific locations.
-                var baseType = BaseTypeNoUseSiteDiagnostics;
-                var interfaces = InterfacesNoUseSiteDiagnostics();
```

Comment:
>var interfaces = InterfacesNoUseSiteDiagnostics(); [](start = 16, length = 50)

I would recommend to keep these locals because I think it would simplify the next merge with master.